content/docs/guides/continuous-delivery/troubleshooting-guide.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Other
+meta_desc: This page walks-through the common failures encountered while running Pulumi in CI/CD, as well as tips on how to fix them.
+menu:
+    userguides:
+        parent: cont_delivery
+        weight: 2
+---
+
+In order to understand the errors encountered during an automated pipeline execution, it is important to understand
+the steps involved in a typical CI/CD configuration for Pulumi regardless of the CI/CD service you are using.
+The type of failure you experience is likely related to one of these steps.
+
+## Overall Requirements
+
+In order to run a Pulumi command, the following are the requirements for any configuration involving Pulumi:
+
+* A Pulumi Access Token for the account you wish to use.
+Create one [here](https://app.pulumi.com/account/tokens) by logging in with the appropriate account.
+* A stack that you would like to update the automated pipeline.
+* Build tools (more on this below) based on the runtime of your Pulumi app.
+* Restore all dependencies for your Pulumi app.
```

Comment:
Perhaps rephrase this to match the rest of the list, which are all things rather than actions?

Reply:
Ah great catch! I'll drop "restore".